hw/ip/otbn/util/otbn-objdump

```diff
@@ -132,7 +132,8 @@ def main() -> int:
             return subprocess.run(cmd).returncode
         else:
             cmd = [objdump, '-M', 'numeric,no-aliases'] + args
-            proc = subprocess.run(cmd, capture_output=True, text=True)
+            proc = subprocess.run(cmd, stdout=subprocess.PIPE,
+                    stderr=subprocess.PIPE, universal_newlines=True)
```

Comment:
Thanks! Please could you use the same formatting as the rest of the file, giving something like this?
```python
            proc = subprocess.run(cmd,
                                  stdout=subprocess.PIPE,
                                  stderr=subprocess.PIPE,
                                  universal_newlines=True)
```
I personally think the weird "drop back a bit" indentation is super-ugly. Since we don't actually mandate it in our style guide, I'm pushing back against it!